packages/cozy-scanner/src/locales/en.json

```diff
@@ -0,0 +1,79 @@
+{
+  "Scan": {
+    "scan_a_doc": "Scan a doc",
+    "save_doc": "Save the doc",
+    "filename": "Filename",
+    "save": "Save",
+    "cancel": "Cancel",
+    "qualify": "Categorize",
+    "apply": "Apply",
+    "error": {
+      "offline": "You are currently offline and you can't use this functionnality. Try it later",
+      "uploading": "You are already uploading a file. Wait until the end of this upload and try again.",
+      "generic": "Something went wrong. Please try again."
+    },
+    "successful": {
+      "qualified_ok": "You just have successfully categorized your file! "
```

Comment:
Sometimes we see qualified, sometimes "categorized", it seems confusing.

Reply:
We now use `describe` in English